libs/features/mas/web-components/src/global.css.js

```diff
@@ -68,6 +70,7 @@ styles.innerHTML = `
     --consonant-merch-card-border-color: #eaeaea;
     --color-accent: #1473E6;
     --merch-color-focus-ring: #1473E6;
+    --merch-color-grey-60: #6D6D6D;
```

Comment:
can we use corresponding spectrum variable and fallback to `#6D6D6D;` ? 
Thus, it should work with dark theme automatically.

Reply:
using the spectrum var inside merch var and the hex value as fallback didn't work. But I changed the reference to use spectrum var and hex as fallback directly in the selectors that use it.